pkg/rootless/rootless_linux.go

```diff
@@ -104,6 +104,9 @@ func BecomeRootInUserNS() (bool, int, error) {
 	var uids, gids []idtools.IDMap
 	username := os.Getenv("USER")
 	mappings, err := idtools.NewIDMappings(username, username)
+	if err != nil && os.Getenv("PODMAN_ALLOW_SINGLE_ID_MAPPING_IN_USERNS") == "" {
```

Comment:
nit: would be nice if the env var was documented (if you mean to support it, if not, feel free to ignore this comment)

Reply:
ah no, this is a hack just to make the CI happy with a single user.  We will need to test the multiple uids/gids feature probably not as part of the integration tests